### DISCUSSION TITLE
Hatósági ár

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#38] [Kérdések és válaszok]
A hatósági ár felhőalapú pénztárgép esetén hogyan értelmezendő? A felhőalapú pénztárgép esetén az “AE-nként” fogalom úgy értendő, hogy ahány eszközre telepítve van epg alkalmazás?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
